### PR TITLE
MAINT: update lfe ppms to match new downstream params

### PR DESCRIPTION
**Pull request by ZLLentz**
+ Set velocities to 12mm/s in epics for min vibration

### CODE REVIEW

**Review by slactjohnson — APPROVED** [review]
I can't comment on the validity of the parameters, but they seem fine to me. 

LGTM other than Travis.

File: plc-lfe-motion/_Config/NC/Axes/Axis 23 IM1L1-PPM-MMS.xti [hunk 1]
```diff
@@ -1469,8 +1467,8 @@ Drive Status 4 (manually linked):
 		</Drive>
 		<Controller Name="Ctrl" CtrType="2">
 			<CtrPara PriorControlFactor="1">
-				<PosDiffControl Range="1"/>
-				<PID PosKp="20" PosExtKp="1" DeadBandPosition="0.002"/>
+				<PosDiffControl Range="10" Time="0.1"/>
+				<PID PosKp="10" PosExtKp="0.01" PosExtVelo="0.0005" DeadBandPosition="0.0001"/>
```
Comment (mkestra) [inline plc-lfe-motion/_Config/NC/Axes/Axis 23 IM1L1-PPM-MMS.xti:1471]: wow, I think this is the first PID loop I've seen her in my  job as a controls engineer!

**Comment by ZLLentz** [comment]
travis here is a pre existing condition

**Review by mkestra — APPROVED** [review]
I agree with Tyler, without knowing exactly how this system is spec'd I can't comment on specific values, but the changes don't seem unreasonable compared to their previous values.

**Review by klauer — COMMENTED** [review]
The updated values appear to be reasonable given the original settings and their orders of magnitude. 💯

**Pull request merged**